config/tips.txt

```diff
@@ -1,92 +1,155 @@
-As a security officer, examining a crew member with a secHUD allows you to set their status to arrest. You can even give a reason for the arrest warrant, as well.
-Clowns are extremely clumsy and will often misfire weapons.
-Hulks can't fire weapons because of their meaty fingers.
-The mime's crayon is edible.
-Shooting lasers at a computer console will destroy it.
-Using ethanol on a piece of paper will remove any writing on it.
-Using an emag on the gibber will allow you to throw people into it!
-Using the comms console with captain level access will allow you to broadcast announcements in BIG, RED LETTERS!
-You can repair cracked windows with a welding tool on help intent.
-Hold alt and left click on an adjacent tile to see its contents in the top right pane.
-While observing or as a ghost, double click on people, bots, or the singularity to follow them.
-Ghosts can click on active teleporters, portals, wormholes, or the gateway to jump to its destination!
-Ghosts can double-click their corpse, or the object containing their corpse (such as the cloning scanner), to re-enter it.
-Ghosts can see inside storage items that have been left on the ground.
-If you click on the command/chat bar to give it full focus you can use control + arrow keys to move the cursor around or scroll through the history of commands. (Control + home/end work too)
-Locked down rogue borgs can be dismantled to return the MMI for placement in a new borg.
-Fore = north, aft = south, starboard = east and port = west.
-Many things that bind you can be resisted out of with the resist button. This includes locked or welded lockers, chairs and handcuffs. Whenever stuck, just give it a try!
-You can use . or # instead of : for radio channels. Building this habit reduces the chance of accidentally saying things on common.
-While pulling something, clicking on a tile with an empty hand will move it to that tile. No more getting stuck in maint while dragging something!
-Things that override clicks like decks of cards or paper bins can still be picked up by dragging them to your character. Equipped things that override click can be unequipped by dragging it to an empty hand.
-In the job selection menu, from the lobby, you can use right click on the "Low/Medium/High" priority button to make it cycle backwards.
-If you stay out of camera range for 10 seconds, the AI's track on you will break.
-You can shoot cameras with bullets to disable them remotely.
-Disabling a camera with wirecutters will not cause a camera alarm, but bludgeoning it will.
-Gangs derive their power from their presence and notoriety. Cleaning up gang tags is the best way to slow their growth.
+Glass shards can be welded to make glass, and metal rods can be welded to make metal. Ores can be welded too, but this takes a lot of fuel.
+If you need to drag multiple people either to safety or to space, bring a locker over and stuff them all in before hauling them off.

```

Comment:
Maybe you should mention that then because I've run into that three body limit myself numerous times and it's not particularly intuitive. Just a suggestion.